<launch>
    <node name="keyboard_driver" pkg="rr_platform" type="keyboard_driver.py" output="screen">
        <param name="reverse_speed" type="double" value="0.5"/>
        <param name="forward_speed" type="double" value="1.5"/>
        <param name="turn_limit"    type="double" value="0.35"/>
        <param name="speed_delta"   type="double" value="0.07"/>
        <param name="turn_delta"    type="double" value="0.05"/>
        <param name="refresh_rate"  type="double" value="10"/>
        <param name="speed_decay"   type="double" value="0.95"/>
        <param name="turn_decay"    type="double" value="0.85"/>
    </node>
</launch>
